### PR TITLE
{bio}[intel/2017b] PTESFinder v1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 3 (1 easyconfigs in this PR)
rescomp1 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/c455b2da772fa2be8634fc934daa8f6a for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
rescomp1 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/c7f9b8893099c0a3f951796257cee36a for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
rescomp1 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/99b6b85551d11d635e6abe89fcffc43a for a full test report.

File: easybuild/easyconfigs/p/PTESFinder/PTESFinder-1-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'Tarball'
+
+name = 'PTESFinder'
+version = '1'
+
+homepage = 'ihttps://sourceforge.net/projects/ptesfinder-v1/'
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PTESFinder/PTESFinder-1-intel-2017b.eb]: @boegel - what's going on with this URL?

Comment (boegel) [reply easybuild/easyconfigs/p/PTESFinder/PTESFinder-1-intel-2017b.eb]: Ah, typo, I'll drop the leading `i`, that's what you're referring to, right?

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**